src/org/jitsi/impl/neomedia/transform/csrc/SsrcTransformEngine.java

```diff
@@ -261,6 +266,17 @@ public RawPacket reverseTransform(RawPacket pkt)
                 csrcAudioLevelDispatcher.addLevels(levels, pkt.getTimestamp());
             }
         }
+
+        SequenceNumberRewriter rewriter = ssrcToRewriter.get(pkt.getSSRCAsLong());
```

Comment:
@gpolitis The fact that the rewriter is accessed after the lastN decision means that not all packets will reach here (if write is "false" we never get here) Rewriter needs to see all the packets of all ssrcs.